skbeam/core/accumulators/binned_statistic.py

```diff
@@ -434,36 +458,26 @@ def __init__(self, rbins, phibins, rowsize, colsize,
                 will be called on the values in each bin.  Empty bins will be
                 represented by function([]), or NaN if this returns an error.
         """
+        if origin is None:
+            origin = shape[0]//2, shape[1]//2
```

Comment:
sure then, let's do it before it becomes used routinely. If you could submit that would be great!
correction, I also meant (if index refers to pixel center for 2 based indexing, then true center should be this):
```python
origin = (shape[0]-1)/2, (shape[1]-1)/2
```